smtp/main.go

```diff
@@ -145,7 +151,7 @@ func (s *smtpNotifier) sendSMTPNotification(build *cbpb.Build) error {
 	addr := fmt.Sprintf("%s:%s", s.mcfg.server, s.mcfg.port)
 	auth := smtp.PlainAuth("", s.mcfg.sender, s.mcfg.password, s.mcfg.server)
 
-	if err = smtp.SendMail(addr, auth, s.mcfg.sender, s.mcfg.recipients, []byte(email)); err != nil {
+	if err = smtp.SendMail(addr, auth, s.mcfg.from, s.mcfg.recipients, []byte(email)); err != nil {
```

Comment:
Do you need to add a `Sender` field to the `header` below?

Reply:
From my understanding, mailgun's definition states that only the receiving email server sees the sender domain while the recipients see the from address, unless we want to explicitly add the sender domain.

Reply:
https://tools.ietf.org/html/rfc2822#section-3.6.2 says that if sender and from are different, we should use both headers.

Reply:
resolved!